scripts/release/mule/test/tests/post/verify_genesis_file.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "[$0] Testing network string in genesis.json"
+
+# We're looking for a line that looks like the following:
+#
+#       "network": "mainnet",
+#
+
+cd "./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
+
+if [ ! -f /var/lib/algorand/genesis.json ]
+then
+    echo "[$0] The genesis file is not present."
+    exit 1
+fi
+
+if [ "$NETWORK" != "$(jq -r '.network' /var/lib/algorand/genesis.json)" ]
+then
+    echo "[$0] The network \`$NETWORK\` set in \`genesis.json\` is incorrect."
```

Comment:
This message is confusing.  Please update this message to include the expected and actual network values.  Consider showing the full path of the genesis.json file.

Reply:
I'll update to show the full path.